src/test/groovy/com/synacy/gradprogram/spock/exercise/DeliveryServiceSpec.groovy

```diff
@@ -0,0 +1,41 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+
+class DeliveryServiceSpec extends Specification {
+    DeliveryRequestRepository deliveryRequestRepository
+    DateUtils dateUtils
+
+    DeliveryService deliveryService
+
+    void setup() {
+        deliveryRequestRepository = Mock()
+        dateUtils = Mock()
+        deliveryService = new DeliveryService(dateUtils, deliveryRequestRepository)
+    }
+
+    def "createDeliveryRequest should save delivery request with the valid order details"() {
```

Comment:
We could probably improve the `valid order details` part :+1: Maybe even just having something like `with the correct details`would be more appropriate.